jscomp/main/melc.ml

```diff
@@ -271,14 +271,14 @@ let main: Melc_cli.t -> _ Cmdliner.Term.ret
     | Some bs_module_type, [] ->
       let suffix = match output_name with
         | Some output_name ->
-          let ext = Filename.extension output_name in
```

Comment:
The issue was that `Filename.extension` will return `.js` when passing `foo.bs.js` as input right?

Reply:
yup. we want all the extensions